Dockerfile

```diff
@@ -3,7 +3,6 @@ COPY requirements.txt /requirements.txt
 RUN python -m pip install -U pip
 RUN python -m pip install -r requirements.txt
 RUN apt-get update && apt-get install -y postgresql postgresql-contrib
```

Comment:
In theory we were installing PostgreSQL because `pyscopg2` (PostgreSQL driver for Python) needs one or other library that comes with PostgreSQL. **If** this `apt-get` is coming **after** `pip` this might not be necessary anymore.

**TLDR** IMHO we can get rid of line 5